plugin/wavesurfer.regions.js

```diff
@@ -441,3 +459,8 @@ WaveSurfer.enableDragSelection = function (options) {
     this.initRegions();
     this.regions.enableDragSelection(options);
 };
+
+WaveSurfer.disableDragSelection = function () {
+    this.clearRegions();
```

Comment:
I'm also not sure we should call `clearRegions()` from this method. I suggest it should just do what it says in the name. `clearRegions` can be called separately if needed.